tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000007` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x00000000000000000000000020044a0d104e9e788a0c984a2b7eae615afd046b` <br/>
+  **Meaning:** Sets `proposer` at slot `0x07` (7). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L59-L63). This value matches the current address found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "PROPOSER()(address)" -r https://ethereum-sepolia.publicnode.com`.
+
+todo: this one differs from the template as well, the finalizationPeriodSeconds is set to 12 (0xc) on Base Sepolia, the template said: Units are in seconds, so the value should be `0x93a80` which is 604,800 in decimal.
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000008` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000000000000c` <br/>
+  **Meaning:** Sets `finalizationPeriodSeconds` at slot `0x08` (8). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L66-L70). Units are in seconds, so the value should be `0xc` which is 12 in decimal. This value matches the current address found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "FINALIZATION_PERIOD_SECONDS()(uint256)" -r https://ethereum-sepolia.publicnode.com`
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x000000000000000000000000afeac3ccabcbcb93e0d04fb0337b519360e898b8` <br/>
+  **After:** `0x000000000000000000000000f243bed163251380e78068d317ae10f26042b292` <br/>
+  **Meaning:** Implementation address is set to the new `L2OutputOracle`. The address can be found in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/implementations/networks/sepolia.yaml#L8).
+
+### `0x49Ff2C4be882298e8CA7DeCD195c207c42B45F66` (`OptimismMintableERC20FactoryProxy`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0x49Ff2C4be882298e8CA7DeCD195c207c42B45F66)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/extra/addresses/sepolia/metal.json#L7)
+
+State Changes:
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x00000000000000000000fd0bf71f60660e2f608ed56e1659c450eb1131200002` <br/>
```

Comment:
Similar comment about prior value being an address, which is surprising and unlike the prior upgrades, so let's check this one out also

Reply:
Looks like spacers were added to `packages/contracts-bedrock/src/universal/OptimismMintableERC20Factory.sol` from the deployment to MCP [comparison](https://github.com/ethereum-optimism/optimism/compare/a7ff5a811612fa338d0a6d6dd72dc2ec9badef6d..e6ef3a900c42c8722e72c2e2314027f85d12ced5).

Reply:
Only additions, so it looks safe